Sources/Validation/Convenience/IPAddress.swift

```diff
@@ -0,0 +1,25 @@
+private let ipv4Pattern = "(([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\.){3}([01]?\\d\\d?|2[0-4]\\d|25[0-5])"
```

Comment:
`private` -> `fileprivate`. This allows swift to better optimise and inline this.